### DISCUSSION TITLE
How to switch the language of the connect wallet?

### DISCUSSION THREAD

**Discussion by GalaxySciTech** [rainbow-me/rainbowkit#1070] [Feedback]
How to switch the language of the connect wallet?
<img width="811" alt="image" src="https://user-images.githubusercontent.com/30950645/224224794-c16e318f-0276-456a-99e2-08c8a5cd2915.png">

**Comment by DanielSinclair** [comment]
@GalaxySciTech This is now available with [v1.1.0](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%401.1.0). Reference the docs [here](https://www.rainbowkit.com/docs/localization) on best practices. RainbowKit will detect a user's preferred language automatically, or you can pass in a locale from either your localized router or an drop-down selection for your users.
